src/runtime_src/core/tools/xbmgmt2/SubCmdProgram.cpp

```diff
@@ -434,16 +436,17 @@ auto_flash(xrt_core::device_collection& deviceCollection)
 
     // Perform DSA and BMC updating
     for (auto& p : boardsToUpdate) {
-      bool reboot;
+      bool reboot, warm_reboot;
```

Comment:
While:
`bool reboot, warm_reboot = false;  `
is valid, for readability and better refactoring support,  it is better to declare each variable uniquely.  For example:
```
bool reboot = false;
bool warm_reboot = false;
```